src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days * 40 >= 280) {
+    return (days * 40) - 50;
+  } else if (days * 40 >= 120) {
```

Comment:
If you write `return` inside `if` block, using `else` after that doesn't make sense.
Structure should be like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```

Reply:
thank you for your remarks. I've fixed the code